packages/theme/src/bootstrap-sass/overrides/_navs.scss

```diff
@@ -144,48 +144,72 @@
     // Make the list-items overlay the bottom border
     margin-bottom: -1px;
     margin-right: $mdlg;
+    padding-bottom: $xs;
 
     @media (min-width: $screen-xs) {
       margin-right: $lg;
     }
 
+    &:after {
```

Comment:
`&::` here and on line 163 as well. 

Again, not a huge deal, but we're using `::after` in this file already, we may as well keep it consistent.